.vscode/launch.json

```diff
@@ -0,0 +1,83 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Test Plugin",
+      "runtimeExecutable": "npm",
+      "runtimeArgs": [
+        "run-script",
+        "tdd",
+        "${input:pluginName}"
```

Comment:
Should this be the full path of the file to avoid conflicts with the script since it also sets the `PLUGINS` environment variable? Otherwise it should be an argument so that the script uses it for the variable.

Reply:
Good catch! I'll remove the env var, it's unnecessary.